CONFIG_GUIDE.md

```diff
@@ -36,12 +35,14 @@ Used to define GPU models and specifications:
   {
     "name": "RTX 4090",
     "memory_gb": 24,
+    "memory_gib": 22.35,
```

Comment:
joe, thx your great effort for updating the gib memory, but shall we use a more robust way like applying a conversion rule? 
1GiB = 1024^3 B
1GB = 1000^3 B
therefore the ratio is 0.931322575, equal to 
1 GB = 0.931322575 GiB
We can keep the current configuration this time, but considering the growing list of gpu machines, maybe we can use conversion rule for the future.